CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 ## UNRELEASED
 
+Improvements:
```

Comment:
I think we need a BREAKING CHANGES section and some of your notes need to be moved here:
```
BREAKING CHANGES:

* Connect Injector
  * New flags `-allow-k8s-namespace` and `-deny-k8s-namespace` were added. Previously the injector would inject all namespaces. To maintain that behaviour, set `-allow-k8s-namespace='*'`. If no `-allow-k8s-namespace` flag is set, no namespaces will be injected.
* Catalog Sync
  * `kube-system` and `kube-public` namespaces are now synced from **unless** `-deny-k8s-namespace=kube-system -deny-k8s-namespace=kube-public` are passed to the `sync-catalog` command.
  * Previously, multiple sync processes could be run in the same Kubernetes cluster with different source Kubernetes namespaces and the same `-consul-k8s-tag`. This is no longer possible. The sync processes will now delete one-another's registrations. To continue running multiple sync processes, each process must be passed a different `-consul-k8s-tag` flag.
  * Previously, catalog sync would delete services tagged with `-consul-k8s-tag` (defaults to `k8s`) that were registered out-of-band, i.e. not by the sync process itself. It would delete services regardless of which node they were registered on. Now the sync process will only delete those services not registered by itself if they are on the `k8s-sync` node (the synthetic node created by the catalog sync process)`.
```